app/templates/crate/version.hbs

```diff
@@ -99,7 +97,9 @@
                 </div>
             {{/if}}
         {{/if}}
+        {{/if}}
```

Comment:
Looks like indentation might be off here

Reply:
The indendation was inconsistent to begin with, so I prioritized diff readability. I fixed the indentation now – please take another look.